### DISCUSSION TITLE
Shaders.

### DISCUSSION THREAD

**Discussion by frank038** [WayfireWM/wayfire#2538] [General]
I've read that shaders are supported by Wayfire. I'd like to know whether this is true or not, and I'd also like to know how to use them.
I mean using shaders to change the colour of the whole screen, not the colour of selected windows.
I've been using Wayfire version 0.9.0.

**Comment by ammen99** [comment]
You will need to compile and install the plugin here: https://github.com/soreau/filters

Check out the README of that plugin, it explains how to setup and use the plugin too.

**Reply by frank038** [reply]
Thank you.
